react/src/base/inputs/SprkErrorContainer/SprkErrorContainer.js

```diff
@@ -8,36 +8,43 @@ const SprkErrorContainer = ({
   analyticsString,
   idString,
   message,
-  id }) => (
-    <div
-      className={classNames('sprk-b-ErrorContainer', additionalClasses)}
-      data-analytics={analyticsString}
-      data-id={idString}
-      id={id}
-    >
-      <SprkIcon
-        additionalClasses="sprk-b-ErrorIcon"
-        iconName="exclamation-filled"
-      />
-      <div className="sprk-b-ErrorText">{message}</div>
-    </div>
+  id,
+}) => (
```

Comment:
ESLINT fixes for this component. The rest of error container work is in other issue.